packages/client/__mocks__/fileMock.js

```diff
@@ -0,0 +1 @@
+module.exports = 'test-file-stub'
```

Comment:
Unnecessary -- if a test needs this, it should be a mock local to that test file